File: Python_Programming/Day_5/program37.py
'''
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    IInput : 4  4
    Output : * * * *
             # # # #
             * * * * 
             # # # # 
'''
def Display(Row,Col):

    for i in range(1 , Row+1 , 1):
        for j in range(1 , Col+1 , 1):
            if(i % 2 != 0):
                print("*\t" , end="")
            else:
                print("#\t" , end="")
        print('\n')



def main():
    Row = 0
    Col = 0

    print("Enter the number of rows : ",end = "")
    Row = int(input())

    print("Enter the number of columns : ",end = "")
    Col = int(input())

    Display(Row , Col)

if __name__ == "__main__":
    main()

File: Java_Programming/Day_5/program37.java
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4  4
    Output : * * * *
             # # # #
             * * * * 
             # # # # 
*/

import java.util.*;

class Pattern
{
    public int iRow;
    public int iCol;

    Pattern(int iRow , int iCol)
    {
        this.iRow = iRow;
        this.iCol = iCol;
    }

    public void Display()
    {
        int i = 0;
        int j = 0;

        for(i = 1 ; i <= iRow ; i++)
        {
            for(j = 1 ; j <= iCol ; j++)
            {
                if(i % 2 != 0)
                {
                    System.out.print("*"+"\t");
                }
                else
                {
                    System.out.print("#"+"\t");
                }
            }
            System.out.println();
        }
    }
}

class program37
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the number of rows : ");
        int iRow = sobj.nextInt();

        System.out.print("Enter the number of columns : ");
        int iCol = sobj.nextInt();

        Pattern pobj = new Pattern(iRow , iCol);

        pobj.Display();

        sobj.close();
    }
}